README.md

```diff
@@ -1 +1,53 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Portfolio
+> ![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
```

Comment:
- [ ] For this and the next projects you should add a better description, just try to explain what you did, what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your project description in your README file😉).